packages/lesswrong/styles/main.scss

```diff
@@ -138,6 +138,14 @@ then moved to more specific files as it becomes necessary.
   }
 }
 
+.users-menu-contents a {
+  // When the user menu is first opened, it incorrectly detects the first menu
+  // item as hovered, but doesn't activate it like it would with a real hover.
+  // With Bootstrap's default hover effect for <a> links, this would look like
+  // it's grayed out.
+  opacity: 1 !important;
```

Comment:
Urgh, that does seem like it would block a bunch of other interactions. But might still be the correct call.

Reply:
Note: we just got rid of bootstrap, not sure how that may interface with this.